app/views/api/services/forms/_usage_rules.html.slim

```diff
@@ -0,0 +1,55 @@
+= form.inputs current_account.multiple_applications_allowed? ? 'Application Requirements' : 'Signup & Use' do
+
+  - unless current_account.multiple_applications_allowed?
+    // TODO remove this and make it an extra_field? Or how about just remove it?
+    = form.input :intentions_required
+
+  = form.input :buyers_manage_apps
+
+  - if @service.backend_version.app_keys_allowed?
+    = form.input :buyers_manage_keys
+
+  = form.input :referrer_filters_required, as: :boolean
+
+  = form.input :custom_keys_enabled
+
+  - if @service.backend_version == "2"
```

Comment:
not sure if it should be `- if @service.backend_version.app_keys_allowed?` :thinking:

Reply:
I didn't want to change much, just move the file around :)